src/NSwag.Sample.NETCore11/Controllers/PetController.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+
+namespace NSwag.Sample.NETCore11.Controllers
+{
+    [Route("pet")]
+    public class PetController : Controller
```

Comment:
Took @rynowak's sample from https://github.com/rynowak/KodKod with some small tweaks to cover a couple more test scenarios